namespaces/ovirt.yml

```diff
@@ -0,0 +1,27 @@
+
+namespace:
+  name: ovirt
+  type: group
+  description: >
+    Namespace for ovirt metadata.
+  fields:
+  - name: entity
+    type: string
+    description: >
+      The type of the data source, hosts, vms, engine etc.
+
+  - name: host_id
+    type: string
+    description: >
+      The oVirt host UUID.
+
+  - name: engine
+    type: group
```

Comment:
Why not add other engine field names later?  This is a schema of sorts, but does not record all the fields that have to be there, or all the field names that can only be there.

The purpose is to define the data type of a field when it *_is_* present.  So if new fields show up later, we can add definitions here as necessary.  If future proof is just adding that field into the hierarchy namespace of `ovirt.engine...`.  Does that makes sense?